app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] The comment belongs to the author, kindly consider implementing this so the relations are setup correctly
e.g ``belongs_to :author, class_name: 'User'``